ci/docker/x86_64-unknown-linux-musl/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM ubuntu:18.04
+
+RUN apt-get update && apt-get install -y \
+  musl-dev \
+  musl-tools \
+  curl \
+  ca-certificates \
+  perl \
+  make \
+  gcc
```

Comment:
Install sccache here: https://github.com/mozilla/sccache/releases/tag/0.2.8
Just download and extract it in PATH